### PR TITLE
eCR Library Table Formatting Part 1

### LINKED ISSUE #2152: eCR Library Table Formatting | Non-Integrated eCR Viewer
## User Story

As somebody viewing the eCR Library table, I want to see a user-friendly table, so that I can easily understand and scan through the eCRs my public health department has received

## Acceptance Criteria

The table must match the Figma designs linked below, specifically:
- [x] There should be a drop shadow below the table header (Y offset: -4px, Blur: 4px, Color: 000000 10% opacity)
- [x] Alternating grey and white rows, making sure colors in table match Figma designs
- [x] The table should have rounded corners (4px)
- [ ] The column widths should look visually similar to the designs. There should be a minimum width on each column such that the column header stays on one line when making the table narrower. If the user makes the browser narrower than the min width, the table should not get any narrower (the user would have to just scroll right and left on the page to see the full table) 
- [x] The table should be responsive to resize when the browser is resized, so the padding around the table should be held constant
- [x] Set table height so that scrolling is inside the table, not the whole page

## Scenarios

_Use the scenarios below to help in testing the ticket before/during/after design review to ensure it meets the criteria above_

Scenario 1:
- I resize my browser window to be wider and taller
- The table smoothly resizes to match the browser window, maintaining constant padding between the table and the edges of the page

Scenario 2:
- I resize my browser window to be narrower
- The columns all decrease in width, until they reach their minimum width
- All column headers stay on one line
- When I make it super narrow, the browser forces me to scroll right and left to see the full table

Scenario 3:
- I try to scroll while hovering outside of the table
- I should not be able to scroll
- I try scrolling within the table area, below the table header
- I am able to scroll through all of the eCRs, and I see a scroll bar to indicate how far I am in the results
- I also see that every row alternates in background color between white and light gray

## Design Notes

[Link to Figma design in Dev mode](https://www.figma.com/design/tXWUeV3Kjehd7VNqHojZYq/eCR-Viewer?node-id=4035-11612&m=dev&t=9byjUxL08W4VVheL-1)
[Link to Figma prototype](https://www.figma.com/proto/tXWUeV3Kjehd7VNqHojZYq/eCR-Viewer?page-id=3953%3A22895&node-id=4035-11612&viewport=-106%2C-192%2C0.65&t=3mxfC0KjphQboN98-1&scaling=contain&content-scaling=fixed&starting-point-node-id=4035%3A11612)

_Screenshots below should not be used for working on this ticket - always refer to Figma design for most up-to-date designs:_

Focus of this ticket is just on styling for the highlighted red area. not worrying about pagination components, header, page title, or search at this time (and checking the actual contents of the table is a different ticket):

### PR DESCRIPTION
**Pull request by angelathe**
# PULL REQUEST

## Summary
**Make initial design changes to eCR library page.** (NOTE: See Additional Info section below).
- Add drop shadow below header
- Add striped table rows, starting with white and alternating to gray
- Responsive to table/browser resize & constant padding
- Table scrolling within table, not the page

**Didn't cover:**
- Min/max widths for overall table
- Min widths for individual columns (so column names don't wrap onto multiple lines)

## Related Issue
Fixes #2152 

## Additional Information
**Important!** The designs for which this ticket was written have been updated. Agreed with Sarah to pause work on this ticket -- will finish/update the library page designs in the new ticket (#2244).

## Screenshots
https://github.com/user-attachments/assets/288d3906-b05f-46c8-b854-e6444a13552a

## Checklist

- [ ] If this code affects the other scrum team, have they been notified? (In Slack, as reviewers, etc.)

[//]: # (PR title: Remember to name your PR descriptively!)

### CODE REVIEW

**Comment by sarahtress** [comment]
you can skip my review on this one! I'll just review the second ticket in the series!

File: containers/ecr-viewer/src/styles/custom-styles.scss [hunk 1]
```diff
@@ -140,15 +140,53 @@ h4 {
...
+  border: 1px solid var(--ink, #111);
+  border-collapse: separate;
+  border-radius: 4px !important;
+  overflow: hidden !important;
+
   font-size: 16px !important;
+
   thead th {
-    background-color: white !important;
+    background-color: color("base-lighter") !important;
```
Comment (BobanL) [inline containers/ecr-viewer/src/styles/custom-styles.scss:153]: I just tested locally and I don't think we need `!important` anywhere here. I could be wrong, but could you double check?

Comment (angelathe) [reply containers/ecr-viewer/src/styles/custom-styles.scss]: Ah yes, you're right! Thanks for finding, just made the change

**Comment by angelathe** [comment]
FYI, I'm closing the PR and issue because the designs have changed. These changes will be carried over into #2244.

**Pull request closed without merging**